app/createTournament/createTournamentDialog.js

```diff
@@ -0,0 +1,21 @@
+import * as tournamentValidation from './validateTournament.js';
+import * as tournamentBusiness from '../tournamentBusiness.js'
+function showOrHideErrorMessage(validationResult) {
+	var $errorSummary = $('#errorsummary');
+	if (validationResult.error===null) {
+		$errorSummary.hide('slow');
+	} else {
+		var message = tournamentValidation.getErrorMessage(validationResult.error);
+		$errorSummary[0].innerText = message;
+		$errorSummary.show('slow');
+	}
+}
+export function createTournamentDialog(){
+ var listofPlayers=tournamentBusiness.getListofPlayers();
```

Comment:
Don't use global variables. See that at the end of the day this players list is global. Instead, use UI.js to inject it to modules. Thus we avoid global dependencies.